src/arrow.js

```diff
@@ -93,4 +96,23 @@ export default class Arrow {
         this.calculate_path();
         this.element.setAttribute('d', this.path);
     }
+    
+    //  add event handling for Arrows
+    setup_eventListener(){
+        $.on(this.element, 'click', e => {
+        	//  remove Arrow element, and delete dependency from task
+        	var index = this.to_task.task.dependencies.indexOf(this.from_task.task.id);
+        	this.to_task.task.dependencies.splice(index, 1);
+        	this.element.remove();
+        	this.gantt.setup_dependencies();
+			// fire dependencyAdded event
+			this.gantt.trigger_event('dependency_removed', [this.to_task.task]);
```

Comment:
Lot of indenting issues. Please run `yarn run prettier` to fix formatting issues.